backend/canisters/community/impl/src/updates/end_video_call.rs

```diff
@@ -0,0 +1,25 @@
+use crate::guards::caller_is_video_call_operator;
+use crate::{mutate_state, run_regular_jobs, RuntimeState};
+use canister_tracing_macros::trace;
+use chat_events::EndVideoCallResult;
+use community_canister::end_video_call::{Response::*, *};
+use ic_cdk_macros::update;
+
+#[update(guard = "caller_is_video_call_operator")]
+#[trace]
+fn end_video_call(args: Args) -> Response {
+    run_regular_jobs();
+
+    mutate_state(|state| end_video_call_impl(args, state))
+}
+
+fn end_video_call_impl(args: Args, state: &mut RuntimeState) -> Response {
+    if let Some(channel) = state.data.channels.get_mut(&args.channel_id) {
+        let result = channel.chat.events.end_video_call(args.message_index, state.env.now());
+        if matches!(result, EndVideoCallResult::Success) {
```

Comment:
Activity notification?

Reply:
I figured the bridge ending a video call didn't necessarily indicate communitiy/group activity